wifimgr.py

```diff
@@ -119,62 +222,58 @@ def handle_root(client):
     wlan_sta.active(True)
     ssids = sorted(ssid.decode('utf-8') for ssid, *_ in wlan_sta.scan())
     send_header(client)
+    client.sendall("<html>")
+    client.sendall(html_head)
     client.sendall("""\
-        <html>
-            <h1 style="color: #5e9ca0; text-align: center;">
-                <span style="color: #ff0000;">
-                    Wi-Fi Client Setup
-                </span>
-            </h1>
-            <form action="configure" method="post">
-                <table style="margin-left: auto; margin-right: auto;">
-                    <tbody>
+            <body>
+                <form action="configure" method="post">
+                    <h1>
+                        Wi-Fi Client Setup
+                    </h1>
+                    <h3>
```

Comment:
you have a h3 after a h1 here. use h2?

Reply:
`h3` is smaller than `h2` and why write extra CSS to make `h2` smaller than one can just use `h3` ?

Reply:
usually the nesting of headlines follows some logic and inside a h1, the next logical thing is h2.

at some other place, we also have that h2 inside/after h1, just not here.

so it seemed to me like some structural / nesting issue.